ait/commons/util/command/upload.py

```diff
@@ -54,73 +105,37 @@ def get_files(upload_path, curr_path, level):
                         # skip hidden files and dirs
                         if not exclude(f):
                             if os.path.isfile(full_path):
-                                f_size = os.path.getsize(full_path)
-                                rel_path = full_path.replace(upload_path + ('' if upload_path.endswith('/') else '/'), '')
-                                fs.append(FileTransfer(path=full_path, key=rel_path, size=f_size))
+                                files.append(full_path)
 
                             elif os.path.isdir(full_path):
                                 get_files(upload_path, full_path, level)
 
             for p in ps:
                 if os.path.isfile(p):  # explicitly specified files, whether hidden or starts with '__' not skipped
-                    f_size = os.path.getsize(p)
-                    f_name = os.path.basename(p)
-                    fs.append(FileTransfer(path=p, key=f_name, size=f_size))
+                    files.append(p)
 
                 elif os.path.isdir(p):  # recursively handle dir upload
```

Comment:
Have you considered [os.walk](https://docs.python.org/3.7/library/os.html#os.walk)?

Reply:
Good idea. I hadn't refactored this part of the code

Reply:
leaving this bit as is for now, as it will be taken care of in this ticket: https://app.zenhub.com/workspaces/dcp-ingest-product-development-5f71ca62a3cb47326bdc1b5c/issues/ebi-ait/dcp-ingest-central/216